### PR TITLE
GCI-41 Edit Account Details Widget

### PR DESCRIPTION
**Pull request by chaityabshah**
[Melange Task](https://www.google-melange.com/gci/task/view/google/gci2014/4897750755835904)
[Ticket](https://issues.openmrs.org/browse/GCI-41)
@elliottwilliams I applied the suggestions you had from GCI-38 (primary colored buttons). How does it look?

### CODE REVIEW

**Comment by elliottwilliams** [comment]
Good start! Here are my suggestions:
- Don't use the primary color here, use `success` for these buttons. The design language I've tried to use in other areas of the dashboard (which, I admit, should be documented in the style guide), is that green buttons do the "save" or "submit" action that corresponds to the main content on the page. In this case, since the Edit Account page's main content is information about the user's account, the buttons should be green, `success` style. (Does this make sense why in the notification I suggested to use primary, not success? :wink: )
- The username at the top of the widget should be the user's username in large text, and the label "username" below it. Right now, it's showing my first name in large text, and my username in label text:
  screen shot 2015-01-02 at 13 30 50
  The purpose of this field is to show users what their username is.
- Can you collapse the password change form, so that the form is only revealed when "Need to change your password?" is clicked. This is the way it's done in the wireframes and Hannah and I thought it would help reduce clutter on this screen.

Thanks as always!

**Comment by chaityabshah** [comment]
@elliottwilliams Added the things you suggested :)

File: app/system-modules/profile/views/details.ejs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="panel panel-default" style="padding: 10px">
+  <h3> <%= user.firstName  %> <small> <%= user.username  %></small></h3>
```
Comment (elliottwilliams) [inline app/system-modules/profile/views/details.ejs]: Like we discussed, this should be the user's username in the h3, then the word _username_ in the small.

File: app/system-modules/profile/views/details.ejs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        <input class="btn btn-primary" type="submit" value="Update Name &#187;">
+    </div>
+</form>
+  <hr>
+  <strong>Your profile picture</strong>
+  <p>Profile pictures are managed along with your profile on the OpenMRS Wiki. 
+  <br>
+  <a href="https://wiki.openmrs.org/users/editmyprofilepicture.action" target="_blank">Edit your profile picture in the wiki</a>.</p>
+  <hr>
+  <strong> Need to change your password?</strong>
```
Comment (elliottwilliams) [inline app/system-modules/profile/views/details.ejs]: Shouldn't this be a link with `id="changePwdLink"`?

**Comment by elliottwilliams** [comment]
Unless something's wrong on my side, I'm seeing some inconsistencies in your design. See my comments in the diff. Also, I would suggest using `display: none` instead of `visibility: hidden` for the password form. (see https://stackoverflow.com/questions/133051/what-is-the-difference-between-visibilityhidden-and-displaynone)

**Comment by elliottwilliams** [comment]
I'm still having problems toggling the password form.
Here's what I would suggest: add a class `change-pwd-visible` that only has one attribute: `display: block;`. Then when that change password link is clicked, use [jQuery's toggleClass method](https://api.jquery.com/toggleclass/) to toggle `change-pwd-visible` on the form. That way you aren't having to mess with CSS style in your JS, which should help clean things up.

**Comment by elliottwilliams** [comment]
Alright, it's working for me too. Thanks!

**Pull request closed without merging**